bootbox.js

```diff
@@ -317,9 +319,15 @@
 
   exports.confirm = function() {
     var options;
+    var invertButtons = defaults.invertButtons;
+    var dialog;
 
     options = mergeDialogOptions("confirm", ["cancel", "confirm"], ["message", "callback"], arguments);
```

Comment:
But the cancel and confirm button would have the active button inverted isn't it? @makeusabrew